beta/kustomize/overlays/workspace/README.md

```diff
@@ -0,0 +1,20 @@
+# Workspace Overlay
+
+This section is only relevant if Google Workspace is your identity provider. 
+
+## Create a Google service account
+
+Create a Google service account and key as outlined in the 1Password support article [Connect Google Workspace to 1Password SCIM Bridge](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client). 
+
+Download the credentials file provided by Google and save a copy to your 1Password account. 
+
+## Create a Kubernetes secret for the Google service account credential
+
+Place the credentials file in the `overlays/workspace` folder. The kustomization overlay will create this as an additional secret.
+
+## Create a Kubernetes secret for the Google Workspace settings file
```

Comment:
Same comment regarding clearing up the wording on this step here to mimic the main readme on on preparing for secret creation